backport/patches/features/eu-debug/0001-drm-xe-Set-XE_BO_FLAG_PINNED-in-migrate-selftest-BOs.patch

```diff
@@ -12,15 +12,16 @@ Reviewed-by: Matthew Auld <matthew.auld@intel.com>
 Link: https://patchwork.freedesktop.org/patch/msgid/20241126174615.2665852-8-matthew.brost@intel.com
```

Comment:
features/eu-debug: Resolves the taint for xe_migrate test


1. update " drm/xe: Set XE_BO_FLAG_PINNED in migrate selftest BOs" Kunit support.
2. add missing patch "drm/xe/display: Update intel_bo_read_from_page to use ttm_bo_access"